application/templates/static_site/_table.html

```diff
@@ -17,6 +14,9 @@ <h3 class="govuk-heading-s">{{ dimension.dimension_table.table_object.header }}<
 
       <div class="table-container">
     <table class="govuk-table govuk-!-margin-bottom-0">
+    {% if dimension.dimension_table.table_object.header %}
+      <caption class="govuk-heading-s eff-table__caption">{{ dimension.dimension_table.table_object.header }}</caption>
```

Comment:
@samuelhwilliams we should use `govuk-table__caption` here instead of `govuk-heading-s` I think – that way the table headings should be left-aligned rather than centred.